src/NuGet.Services.Staging.Manager/V3/StageIndexBuilder.cs

```diff
@@ -7,12 +7,12 @@
 
 namespace NuGet.Services.Staging.Manager.V3
 {
-    public class StageIndexBuilder
+    public static class StageIndexBuilder

```

Comment:
Refactored the class to get IV3ServiceFactory in the ctor, now it makes more sense to keep it non static. IMO, a class that has  no ctor or fields, should be static.

---
In reply to: [65841839](https://github.com/NuGet/NuGet.Staging/pull/58#discussion_r65841839) [](ancestors = 65841839)